model/version/version.go

```diff
@@ -75,3 +76,54 @@ var (
 	BuildStatusActivateAtKey = bsonutil.MustHaveTag(BuildStatus{}, "ActivateAt")
 	BuildStatusBuildIdKey    = bsonutil.MustHaveTag(BuildStatus{}, "BuildId")
 )
+
+type DuplicateVersionsID struct {
+	Hash      string `bson:"hash"`
+	ProjectID string `bson:"project_id"`
+}
+
+type DuplicateVersions struct {
+	ID       DuplicateVersionsID `bson:"_id"`
+	Versions []Version           `bson:"versions"`
+}
+
+func FindDuplicateVersions(since time.Time) ([]DuplicateVersions, error) {
+	pipeline := []bson.M{
+		bson.M{
+			"$match": bson.M{
+				RequesterKey: evergreen.RepotrackerVersionRequester,
```

Comment:
just to confirm, push-event triggered builds still have this requester?

Reply:
Yes. FYI, there are three requesters, repo tracker, patch, and GitHub

Reply:
cool! 

github is used for pull requests only, and the webhook triggered repotracker is still "repotracker" (gitter?)

Reply:
All repotracker requests (polling, and push event triggered) have requester "gitter_request"